config/formats.ts

```diff
@@ -591,37 +591,61 @@ export const Formats: FormatList = [
 		section: "Pet Mods",
 	},
 	{
-		name: "[Gen 8] Fusion Evolution UU",
+		name: "[Gen 8] Alternatium",
+		desc: `<b>Restrictions</b>: A metagame made up of only Pokemon with alternate forms, with all of them being seperate and unique Pokemon.`,
 		threads: [
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3674163/">Fusion Evolution UU</a>`,
-		],
-
-		mod: "fusionevolutionuu",
-		searchShow: false,
-		ruleset: ['Standard NatDex', 'OHKO Clause', 'Evasion Moves Clause', 'Species Clause', 'Dynamax Clause', 'Sleep Clause Mod', 'Z-Move Clause', 'Data Mod', 'Mega Data Mod'],
-		banlist: [
-			'All Pokemon', 'Lopunnite', 'Tapu Lop-Mega', 'Red Orb', 'Grousle-Primal',
+			`&bullet; <a href="https://www.smogon.com/forums/threads/alternatium-slate-7-slow-twins-slate-also-vote-in-poll.3683767/">Alternatium on Smogon Forums</a>`,
+			`&bullet; <a href="https://docs.google.com/spreadsheets/d/1bvvkPg1CrUBJFJJeuwkts8elfJcEcahGOoHm-vGBXOI/edit?usp=sharing">Spreadsheet</a>`,
+			`&bullet; <a href="https://dragonheaven.herokuapp.com/">Server with Teambuilder Support</a>`,
```

Comment:
@Zarel what's your stance on this URL?

Reply:
It'd be a huge help to us since that has a working teambuilder for our mod, and it's very hard to teambuild without it. Dragon Heaven is already linked from Pokemon Showdown in their list of other servers, although that's not a link to the custom client.

Reply:
If we're going to have server-side support for the format, I see no reason why we wouldn't allow adding client support for it too.

Reply:
There would be nothing wrong with adding client-side support on your server, aside from potential difficulty. We would really love to have it. Random Dex would be quite easy, as it's just natdex with a much larger banlist.

Alternatium has a good number of custom moves and abilities, as well as some fake pokemon. The pokemon also have radically different stats, abilities, and movepools. Is it feasible to provide support for that without a huge amount of trouble? I am somewhat familiar with the client, as I made the Dragon Heaven client. My implementation of mod support wasn't really up to your standard, although I imagine this would be implemented much differently (hard-coded like LGPE or other OMs). 

I'd be willing to try my hand at it, although a little bit of guidance from one of the main client coders would go a long way. I know Kris had worked on mod support in the past, so maybe he has some ideas? Of course I wouldn't mind if he or anyone else wanted to implement it, but I know you are all busy, and the Pet Mods community are the main beneficiaries of this (we really appreciate it by the way.)

I removed the link. We'd be fine with this going up as soon as possible, without teambuilder support for now. We are used to recieving people in the Pet Mods room, telling them how to use the Import feature or directing them to Dragon Heaven. Let me know what you think of all this, and what you think the best plan would be. Thanks.

Reply:
It seems like there has been significant progress on client support for mods: https://github.com/smogon/pokemon-showdown-client/pull/1713

Reply:
Is finishing and accepting the PR you linked a realistic possibility? It's been inactive for 9 months. I'd love to see it go through, but I'd rather not delay PMOTS indefinitely if there isn't a lot of interest from the people who can make it happen. I was aware of this work, I just figured since it wasn't accepted I'd look for a less general, hopefully simpler solution for just Alternatium support. Again, I'd like to know what the admins think the best plan would be.